extensions/amp-web-push/validator-amp-web-push.protoascii

```diff
@@ -57,6 +57,13 @@ tags: {  # <amp-web-push>
       allow_relative: false
     }
   }
+  attrs: {
+    name: "service-worker-scope"
+    value_url: {
+      protocol: "https"
+      allow_relative: true
```

Comment:
We will need to update AMP caches to rewrite this attribute from relative to absolute before merging. Filed b/131437166. We could move forward now without relative URLs in this PR and allow relative URLs after that issue is addressed.

Reply:
Context here, this value is only used when the AMP doc is loaded on origin. It won't be used if served from a cache.

Given that, is the rewrite necessary?

Reply:
If the Runtime is guaranteeing that then the rewrite is not necessary.

Reply:
This is a security guarantee by by the browser, you can't register a service worker unless the origin of the SW and the page's origin match. So there's never really a point to use this unless we're on the original page's origin, meaning it'll be the correct relative URL.

Reply:
That makes sense, thank you for explaining.

BTW, `true` is the default value for `allow_relative` so the line is unnecessary.